cdk/lib/admin-console.test.ts

```diff
@@ -5,7 +5,11 @@ import { AdminConsole } from './admin-console';
 describe('The AdminConsole stack', () => {
   it('matches the snapshot', () => {
     const app = new App();
-    const stack = new AdminConsole(app, 'AdminConsole', { stack: 'support', stage: 'TEST' });
+    const stack = new AdminConsole(app, 'AdminConsole', {
+      stack: 'support',
```

Comment:
Minor thing (and personal preference really): I have started pulling my [`PROD` props into a const](https://github.com/guardian/amigo/blob/3f0bb6b1c75d2029eeddac0d6fcc27ee850c75e8/cdk/bin/cdk.ts#L20-L24) and [passing these into the test](https://github.com/guardian/amigo/blob/3f0bb6b1c75d2029eeddac0d6fcc27ee850c75e8/cdk/lib/amigo.test.ts#L9) as well as the [stack instantiation](https://github.com/guardian/amigo/blob/3f0bb6b1c75d2029eeddac0d6fcc27ee850c75e8/cdk/bin/cdk.ts#L26).

This helps to keep your snapshot as close as possible to the template that you will actually upload (i.e. the thing that you really care about not breaking!) - perhaps there are a few subtle bugs that can be caught this way as the number of stage-specific props grows.